filesystem.go

```diff
@@ -58,6 +65,29 @@ func New(fs billy.Filesystem, path string) SivaFS {
 	}
 }
 
+// NewFilesystem creates an entire filesystem using siva as the main backend,
+// but supplying unsupported functionality using as a temporal files backend
+// the main filesystem
```

Comment:
Explain that temporal files will be created in a directory called "/tmp" (which is not a well known path in some OS).